modules/statistics/php/stats_demographic.class.inc

```diff
@@ -163,65 +163,72 @@ class Stats_Demographic extends \NDB_Form
         foreach (\Utility::getProjectList() as $key => $value) {
             $projects[$key] = $value;
         }
-        $currentProject = null;
-        if ($_REQUEST['DemographicProject'] ?? ''
-        ) {
-            $currentProject = htmlspecialchars($_REQUEST['DemographicProject']);
+        $demographicProject = $_REQUEST['DemographicProject'] ?? '';
```

Comment:
`$demographicProject` replaced `$currentProject` but here it is being set outside of the if statement below, previously it's value was null if the IF is not entered `' ' != null` theres a change of behaviour